### ISSUE TITLE
Improve gradient-background

### ISSUE THREAD

**Issue by jpmarques66** [reach4help/reach4help#201]
Gradient should be like the following, to fit better with the logo.
```linear-gradient(135deg,#F27979 0,#7D00A3 80%) ```

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/201)

*This information is updated automatically. To modify it, please use ZenHub.*

**Comment by btuerker** [comment]
Hi @jpmarques66, I converted the linear gradient in SVG file into CSS gradient and found that the gradient in the logo is exactly `linear-gradient(135deg, #F27979 0%, #7D00A3 100%)`.

![Screen Shot 2020-04-05 at 8 44 07 PM](https://user-images.githubusercontent.com/46192266/78505896-1e3baa00-777f-11ea-8f89-af93705c63e6.png)

Here's the original look:
![Screen Shot 2020-04-05 at 8 47 15 PM](https://user-images.githubusercontent.com/46192266/78505911-30b5e380-777f-11ea-9baf-3861916bc315.png)

Then, here's with your value: `linear-gradient(135deg, #F27979 0%, #7D00A3 80%)`
_**Personal opinion:** Purples at the bottom looks a bit intensive to me._
![Screen Shot 2020-04-05 at 8 47 21 PM](https://user-images.githubusercontent.com/46192266/78505928-44614a00-777f-11ea-88f6-7b08d38393e3.png)

Then, here's with the exact same gradient value: `linear-gradient(135deg, #F27979 0%, #7D00A3 100%)`
![Screen Shot 2020-04-05 at 8 47 26 PM](https://user-images.githubusercontent.com/46192266/78505961-5e029180-777f-11ea-8101-55dc0e4ce48c.png)

@jpmarques66 I'll be waiting for you to decide which one we should use? 

It's a small change and I already obtained it. So, after your response, I'm going to open a PR for it.
